core/toolbox/toolbox.js

```diff
@@ -0,0 +1,463 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Toolbox from whence to create blocks.
+ * @author fraser@google.com (Neil Fraser)
+ */
+'use strict';
+
+goog.provide('Blockly.Toolbox');
+
+goog.require('Blockly.ToolboxCategory');
+goog.require('Blockly.ToolboxSeparator');
```

Comment:
require `Blockly.utils.dom`

Reply:
Also use `requireType` for any types you're referencing.

Reply:
done.